src/python/pants/bin/daemon_pants_runner.py

```diff
@@ -25,7 +24,7 @@ class ExclusiveRequestTimeout(Exception):
     """Represents a timeout while waiting for another request to complete."""
 
 
-class DaemonPantsRunner(RawFdRunner):
+class DaemonPantsRunner:
```

Comment:
`RawFdRunner` is a `Protocol`, which means you do not need to subclass it. MyPy only checks that the required interface is implemented.